client/src/containers/Sensor/components/SensorCreation.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react'
+import { apiResponse } from '../../../epics/utils'
+import { Button, Dialog, DialogActions, DialogContent, TextField } from '@material-ui/core'
+import { SensorCreateParams } from '../../../model/sensor'
+
+interface Props {
+  onCancel: () => void
+  onSubmit: (sensorBase: SensorCreateParams) => void
+}
+
+export default function SensorCreation(props: Props) {
+  const { onCancel, onSubmit } = props
+  const [isLoading, setIsLoading] = useState(false)
+  const [sensorBase, setSensorBase] = useState<SensorCreateParams>({
+    display_name: '',
+    type: 'A',
+    extra: '',
+  })
+
+  useEffect(() => {
+    const subscription = apiResponse.subscribe({
+      next: () => setIsLoading(false),
+    })
+
+    return subscription.unsubscribe()
```

Comment:
`return () => subscription.unsubscribe()` is different from `return subscription.unsubscribe()`

First one return an arrow function which will be executed when the effect is leaving
Second one return the result of subscription.unsubscribe() as the function which will be executed when the effect is leaving